### ISSUE TITLE
Incompatible

### ISSUE THREAD

**Issue by Soulstyme** [Grantapher/ValheimPlus#91]
- **OS:** [e.g. Windows Client, Windows Server, or Linux Server]
- **Valheim Game Version:** [e.g. 0.4.11, etc]
- **V+ Mod Version:** [e.g. 0.9, 0.9.2, etc.]

**Describe the bug**

A clear and concise description of what the bug is and how you found it.

**To Reproduce**

Steps for the devs to reproduce the behavior:
![Screenshot (4)](https://github.com/Grantapher/ValheimPlus/assets/88342145/5c215f89-0b85-4c3b-b3af-06f29c0f3768)

**Expected behavior**

**Troubleshooting**

What steps have you taken to try to fix?

**Screenshots**

**Link to [pastebin](https://pastebin.com) containing .cfg configuration if applicable:**

**Additional context**

*Please list any other mods besides V+ that you have installed* here. Add any other context about the problem here.

**Comment by tndrs** [comment]
Happens because of the Valheim Update IronGate pushed out today.
My guess is that we need to wait for a new ValheimPlus release to be built and patch afterwards.


As a workaround, you can rollback to the old_stable version of Valheim by checking Steam Beta section in the games properties.

**Comment by Soulstyme** [comment]
Thank you for your prompt response. The work around is a success.
Hopefully, VPlus will have an update soon.

**Comment by Dombi-Bence** [comment]
This workaround is sadly not possible for the XBox Game Pass version of the game.

**Comment by tndrs** [comment]
Yeah, default_old is the one. That rolls back to the last working version. Mixed them up somehow ;-)

**Comment by misterjoshua** [comment]
> Yeah, default_old is the one. That rolls back to the last working version. Mixed them up somehow ;-)

Thanks. I appreciate you man. It's working for me now, but I took character backups just in case. :)

**Comment by misterjoshua** [comment]
FYI @Grantapher released https://github.com/Grantapher/ValheimPlus/releases/tag/0.9.13.3, which works now with the latest version according to my testing. I had to restart r2modman to see the update on thunderstore.

I had to perform a backup restoration on my @lloesche -based docker server because it automatically updated overnight and wiped out the bottom rows of my expanded chests after Valheim updated but before ValheimPlus was released. This happened because the default ValheimPlus behavior was to abort loading and start the server without the mod when incompatibility was detected, and after this happened, my expanded chest rows were not supported by the server.

I wonder if ValheimPlus should crash the server when there's an incompatibility rather than starting anyway and risking data corruption.

**Comment by Grantapher** [comment]
@misterjoshua I thought chests were only affected if you tried to open them, did simply starting the world affect the chest contents?

**Comment by Grantapher** [comment]
Also confirming, [`0.9.13.3`](https://github.com/Grantapher/ValheimPlus/releases/tag/0.9.13.3) fixes compatibility for `.19`.

**Comment by misterjoshua** [comment]
> @misterjoshua I thought chests were only affected if you tried to open them, did simply starting the world affect the chest contents?

A player logged in during this timeframe, opened chests, and this was the result. The one who logged in didn't realize this incompatibility had occurred because they weren't watching the game log, and the server was happy to admit them.

They also lost the bottom 2 rows of their character inventory, but they were able to restore cloud saves so this wasn't a huge problem.
